akka-http2-support/src/main/scala/akka/http/impl/engine/http2/RequestParsing.scala

```diff
@@ -16,7 +18,7 @@ import scala.collection.immutable.VectorBuilder
  * INTERNAL API
  */
 private[http2] object RequestParsing {
-  def parseRequest(subStream: Http2SubStream): HttpRequest = {
+  def parseRequest(httpHeaderParser: HttpHeaderParser)(subStream: Http2SubStream): HttpRequest = {
```

Comment:
Need to port correctness checks from `HttpMessageParser` (like duplicate `content-length`, etc.)